tests/ltp/ext2fs_tests_passed.txt

```diff
@@ -88,7 +88,7 @@
 /ltp/testcases/kernel/syscalls/mmap/mmap03
 /ltp/testcases/kernel/syscalls/mmap/mmap04
 /ltp/testcases/kernel/syscalls/mmap/mmap09
-/ltp/testcases/kernel/syscalls/mmap/mmap11
+# /ltp/testcases/kernel/syscalls/mmap/mmap11 does not exit threads; gcov segfaults
```

Comment:
Too bad that there's no way to just disable for GCOV, but that's okay.

I guess we could generate the input file from two files: this file and a gcov exclusions file.

Reply:
I think we could leave this right now since there is only one test impacted and it is potentially impacting for Pauls upcoming PR as well, we might have too many specific exclusions

Reply:
this is not related to gcov. It is crashing in my branch also :(